src/vaadin-menu-bar-interactions-mixin.html

```diff
@@ -87,7 +86,7 @@
     }
 
     _getButtonFromEvent(e) {
-      return Array.from(e.composedPath()).filter(el => el.item)[0];
+      return Array.from(e.composedPath()).filter(el => el.localName === 'vaadin-menu-bar-button')[0];
```

Comment:
Why was this changed? All the tests pass fine without this change.

Reply:
Test is failing on IE11 without this update.